x-pack/plugins/security_solution/public/timelines/components/timeline/body/data_driven_columns/index.tsx

```diff
@@ -444,6 +444,13 @@ export const getMappedNonEcsValue = ({
   data: TimelineNonEcsData[];
   fieldName: string;
 }): string[] | undefined => {
+  /*
+   While data _should_ always be defined
+   There is the potential for race conditions where a component using this function
+   is still visible in the UI, while the data has since been removed.
+   To cover all scenarios where this happens we'll check for the presence of data here
+  */
+  // if (!data || data.length === 0) return undefined;
```

Comment:
This references the fix made here: https://github.com/elastic/kibana/pull/133234 which should not be needed anymore once the other changes go in that return an EmptyComponent when data isn't provided. Will remove before merging